docs/source/markdown/options/userns.container.md

```diff
@@ -12,10 +12,11 @@ Rootless user --userns=Key mappings:
 
 Key       | Host User |  Container User
 ----------|---------------|---------------------
-""        |$UID           |0 (Default User account mapped to root user in container.)
-keep-id   |$UID           |$UID (Map user account to same UID within container.)
-auto      |$UID           | nil (Host User UID is not mapped into container.)
-nomap     |$UID           | nil (Host User UID is not mapped into container.)
+""        |$UID         |0 (Default User account mapped to root user in container.)
+keep-id   |$UID         |$UID (Map user account to same UID within container.)
+keep-id:uid=200,gid=210 |$UID| 200:210 (Map user account to specified uid, gid value within container.)
```

Comment:
Maybe make more generic - `uid=$uid,gid=$uid` and `Map UID and GID of user running podman to $uid and $gid in container`

Reply:
That is what --userns=keep-id does by default.
This is more the case of running a container with a builtin user and you want the builtin user to match your UID.